src/pages/comparison/comparison.tsx

```diff
@@ -225,9 +265,16 @@ export const Comparison = () => {
       loadOptions,
     }),
   ];
+
   return (
     <Container layout={layout}>
       <DeckWrapper layout={layout}>
+        <LeftSideToolsPanelWrapper layout={layout}>
```

Comment:
This should be after `DeckGL` and you don't need z-index then

Reply:
Done